spotify/scripts/dscc-scripts.js

```diff
@@ -0,0 +1,79 @@
+const open = require('opn');
+const argparse = require('argparse');
+const packageJson = require('../package.json');
+const cp = require('child_process');
+const chalk = require('chalk');
+const terminalLink = require('terminal-link');
+
+const exec = (command, options) => {
+  return new Promise((resolve, reject) => {
+    const child = cp.exec(command, options, (err, stdout, stderr) => {
```

Comment:
`exec` is a serious code smell, as it's a common source of injection attacks. Even if you think it's okay in this particular case, you should strongly prefer `execFile` unless you have a *really* compelling reason not to.

Reply:
In particular, this function should have this interface:

    await exec(['npx', '@google/clasp', 'deploy', ...], {options}).then(...)

Reply:
The new version uses execa, which is using execFile as suggested here.